requirements.txt

```diff
@@ -7,3 +7,4 @@ drms>=0.5.7
 zeep>=3.4.0
 matplotlib>=3.2.1
 astropy>=4.0
+scikit-learn>=0.23.1
```

Comment:
why do we have a `requirements.txt` file? we shouldn't need it. That should live in the setup configuration. Requirements file are a thing for websites applications :stuck_out_tongue_closed_eyes: